### DISCUSSION TITLE
The dev version is somewhat usable!

### DISCUSSION THREAD

**Discussion by Beakerboy** [Beakerboy/MS-CFB#8] [Announcements]
@sancarn, simple ole files can now be created from a directory tree using this set of scripts. Everything is very pre-alpha, but let me know if you have any suggestions on interface.

**Comment by sancarn** [comment]
Well done, good to know! :)  Dev branch?

**Reply by Beakerboy** [reply]
Yes, let me know if the README should be more clear about anything, or if you think the overall user interface is confusing.

**Reply by Beakerboy** [reply]
I’ve added a new option to the command line:
`python -m ms_cfb -l -f VbaProject.bin` will print information about the file. So far, the OLE version, GUID, and size, location, and dates of the streams in the file.

if you have any OLE files you would like to test, check it out and let me know if you either have difficulty, or find any bugs.
